compiler/model/build-model.ts

```diff
@@ -279,6 +294,15 @@ function compileClassOrInterfaceDeclaration (declaration: ClassDeclaration | Int
       })
     }
 
+    // If the body wasn't set and we have a parent class, then it's a property body with no additional properties
+    if (!bodyWasSet && type.inherits != null) {
+      const parent = type.inherits.type
+      // RequestBase is special as it's a "marker" base class that doesn't imply a property body type. We should get rid of it.
+      if (!(parent.name === 'RequestBase' && parent.namespace === '_types')) {
+        type.body = { kind: 'properties', properties: new Array<model.Property>() }
+      }
+    }
```

Comment:
Should we check the parent body kind? It might happen that the parent has a `value` body instead of  `properties`.

Reply:
This is checked in `validate-model`: https://github.com/elastic/elastic-client-generator/blob/ce700b315a9ba2838beec1fddc2e0cac1ff06abd/compiler/steps/validate-model.ts#L431-L435